concourse/deploy-cf-perftest.yml

```diff
@@ -98,7 +98,7 @@ jobs:
             source:
               repository: cloudfoundry/cf-deployment-concourse-tasks
             version:
-              digest: sha256:90349a43f1a254d9e458a519fe2d3acc5c655d90ac82a418d009c923df0bcc26
+              digest: sha256:e5a5877fa88caa76b3f8dafe7d15111b295a97506780dfe16cbf1e05a3e4fbad
```

Comment:
Do we need a job to update those digests regularly?

Reply:
Possibly yes. The next versions will however come with Ginkgo v2, so we should first migrate our tests.

Reply:
Do we have a particular reason for not expecting that the latest cf-deployment release will be deployable with the latest release of cf-deployment-concourse-tasks running on its latest image?

I'm thinking we might as well just let the pipeline reveal any incompatibilities to us and just fix them as and when they occur. Alternatively, if an actual compatibility matrix exists somewhere (_and we could write something to generate one in the cf-deployment pipeline..._) we could use that instead.